src/main/java/org/primefaces/extensions/component/gchart/GChartRenderer.java

```diff
@@ -74,7 +74,9 @@ protected void encodeScript(final FacesContext context, final GChart chart) thro
                     .attr("language", chart.getLanguage())
                     .attr("width", chart.getWidth())
                     .attr("height", chart.getHeight());
-
+        if (chart.getExtender() != null) {
+            wb.nativeAttr("extender", chart.getExtender());
```

Comment:
If the extender value is just the name of the function, I think you should just use `wb.attr`

Reply:
I believe native attribute is correct because it’s a reference to a JavaScript function.

Reply:
Yeah, I was confused by the attribute documentation (see below).